browser/ui/webui/settings/brave_extensions_manifest_v2_handler.cc

```diff
@@ -203,8 +180,9 @@ void BraveExtensionsManifestV2Handler::EnableExtensionManifestV2(
 
   if (enable) {
     if (!installed) {
-      installer_ = base::MakeRefCounted<ExtensionWebstoreInstaller>(
-          id, profile, web_ui()->GetWebContents(),
+      installer_ = std::make_unique<
+          extensions_mv2::ExtensionManifestV2Installer>(
```

Comment:
it looks like the UI is guarding against this getting toggle on and off while the extension is installing, but I think it would be prudent to have a check somewhere in the cpp code to make sure it can't be called again while the install is still in progress?